### ISSUE TITLE
Test suite

### ISSUE THREAD

**Issue by jayvdb** [dahlia/iterfzf#7]
It would be helpful to start a test suite for this package.

**Comment by dahlia** [comment]
Yeah, it must be helpful we have any types of automated tests; unit tests or functional tests, whatever.  Could anybody help?
